astarte-device-sdk/AstarteDeviceSDK.cpp

```diff
@@ -373,3 +373,21 @@ AstarteDeviceSDK::ConnectionStatus AstarteDeviceSDK::connectionStatus() const
 
     return static_cast<AstarteDeviceSDK::ConnectionStatus>(m_astarteTransport->connectionStatus());
 }
+
+bool AstarteDeviceSDK::connectToBroker()
+{
+  if (!m_astarteTransport) {
```

Comment:
maybe we should add a qDebug().
`qDebug() << "Not yet initialized, cannot connect to broker."`